docs/cloud-tutorials/best-practices.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 8
+title: Best practices
+description: DBOS Cloud Tips
+---
+
+A list of tips for using DBOS Cloud effectively.
+
+## Managing applications
+* Only one request at a time can manipulate an application and you will get an `application is busy` error if you attempt multiple concurrent modifications.
+* Application names are unique within an [organization](#users-and-organizations).
+* Your application is allocated 1GB of disk space, 1CPU and 512MB of RAM.
+
+### Deployment
+* An application must be registered before being deployed. DBOS Cloud CLI handles registration automatically when you deploy an application for the first time.
```

Comment:
We're removing app registration completely from our docs (the other PR). So we don't need this now.

Reply:
I removed it but the API does not work like that and I expect the question to be raised again in the future.